README.md

```diff
@@ -34,14 +34,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the difference between an adaptive website and a fully responsive website?
 
+//In a responsive design, one page with an optimal viewing experince on any device, and in Adaptie its multiple layouts for screen sizes
+
 2. Describe what it means to be mobile first vs desktop first.
 
+//Mobile first focusus on functionality over features first, and adds features as you increase screen size, and in Desktop first you would provide full functionality then "gracefully degrade"
+
```

Comment:
Features vs functionality isn't the difference between mobile and desktop. It's simply what you start with in your CSS and how you make media queries.